asset/css/markbind.css

```diff
@@ -78,12 +78,53 @@ code.hljs.inline {
     background-color: #fff;
 }
 
+/** Holy Grail Layout **/
```

Comment:
Just wondering if the`/**` is intended instead of `/*`? I can't seem to find any other comments that use this convention in the codebase 😅

Reply:
Because it was intended for `/** Holy Grail Layout **/` to be the main section and for `/* Header */`, `/* Footer */` etc to be sub-sections under the main section. 

I agree that it is subtle and not intuitive. Might change the commenting style to something more widely recognised.

Reply:
After digging around, this seems much better :)

```CSS
/**
  * Holy Grail Layout 
  *
  * This section covers the common styles used within Header, Footer and Side Navigation bars
  */